seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -303,33 +300,24 @@
     },
     {
       "id": "587d7b88367417b2b2512b45",
-      "title": "Arrow functions with higher order functions",
+      "title": "Write Arrow Functions with Higher Order Functions",
       "description": [
-        "It's time we see in action how easily arrow function let us write complex data processing code",
-        "It mixes really well with higher order functions, such as map(), filter(), reduce() etc., that takes other functions as inputs for processing collections.",
+        "It's time we see how powerful arrow functions are when processing data.",
+        "Arrow functions work really well with higher order functions, such as <code>map()</code>, <code>filter()</code>, and <code>reduce()</code>, that take other functions as arguments for processing collections of data.",
         "Read the following code:",
-        "<code>FBPosts.</code>",
-        "<code>.filter(function(post){</code>",
-        "<code>    return post.thumbnail !== null && post.shares > 100 && post.likes > 500</code>",
-        "<code>})</code>",
-        "<code>.reduce(function(acc, current){</code>",
-        "<code>    return acc + '<img alt=\"' + current.text + '\">' + current.thumbnail + '</img>'</code>",
-        "<code>})</code>",
-        "We have written this with filter() and reduce() to at least make it somewhat readable. Now compare the version with arrow functions",
-        "<code>FBPosts</code>",
-        "<code>    .filter((post) => post.thumbnail !== null && post.shares > 100 && post.likes > 500)</code>",
-        "<code>    .reduce((acc, current) => (acc + '<img alt=\"' + current.text + '\">' + current.thumbnail + '</img>'))</code>",
-        "This code is more succinct and keeps only the parts of the code where the real processing happen.",
-        "Instructions.",
-        "Use arrow function syntax to compute square of only the positive integers (fractions are not integers) in the array and return the array.",
-        "Note",
-        "Don't forget to use strict mode."
+        "<blockquote>FBPosts.filter(function(post) {\n  return post.thumbnail !== null && post.shares > 100 && post.likes > 500;\n})</blockquote>",
+        "We have written this with <code>filter()</code> to at least make it somewhat readable. Now compare it to the following code which uses arrow function syntax instead:",
+        "<blockquote>FBPosts.filter((post) => post.thumbnail !== null && post.shares > 100 && post.likes > 500)</blockquote>",
+        "This code is more succinct and accomplishes the same task with fewer lines of code.",
+        "<h4>Instructions</h4>",
+        "Use arrow function syntax to compute the square of only the positive integers (fractions are not integers) in the array <code>realNumberArray</code> and store the new array in the variable <code>squaredIntegers</code>.",
+        "<strong>Note:</strong> Don't forget to use strict mode."
```

Comment:
I think notes should not have colons and a line break `<br>` after "Note"

cc/ @HKuz